modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/behavior/impl/MilestoneActivityBehavior.java

```diff
@@ -30,10 +30,12 @@ public class MilestoneActivityBehavior extends CoreCmmnActivityBehavior {
     
     protected Expression milestoneNameExpression;
     protected String milestoneVariable;
-    
-    public MilestoneActivityBehavior(Expression milestoneNameExpression, String milestoneVariable) {
+    protected String businessStatusUpdate;
+
+    public MilestoneActivityBehavior(Expression milestoneNameExpression, String milestoneVariable,String businessStatusUpdate) {
```

Comment:
very, very minor nitpick; missing space
```suggestion
    public MilestoneActivityBehavior(Expression milestoneNameExpression, String milestoneVariable, String businessStatusUpdate) {
```

Reply:
Thanks for your feedback. I thought I fixed it before you can find it. But I was wrong :)